src/mixins/filing-template-mixin.ts

```diff
@@ -402,7 +400,9 @@ export default class FilingTemplateMixin extends DateMixin {
           nrNumber: this.getNameRequestNumber
         },
         nameTranslations: this.getNameTranslations,
-        parties: this.orgPersonsToParties(this.getAddPeopleAndRoleStep.orgPeople)
+        parties: this.orgPersonsToParties(this.getAddPeopleAndRoleStep.orgPeople),
+        offices: undefined, // *** TODO: update
+        contactPoint: undefined // *** TODO: update
```

Comment:
@ketaki-deodhar ^^^

Reply:
Got it. I will keep an eye on this while merging :)